### PR TITLE
Add postgres wallet support to agent, and merge recent orgbook updates

### PR DESCRIPTION
**Pull request by ianco**

### CODE REVIEW

**Comment by WadeBarnes** [comment]
I'm seeing `.gitignore` files at multiple levels.  Any reason why we would not want to deal with ignores at the top level?

Documentation such as [starter-kits/credential-registry/server/README.md](https://github.com/bcgov/indy-catalyst/pull/228/files#diff-ed7a4e3b83c0bd627198a52d96ff5711) are referencing the existing OrgBook docs/code.  I'm assuming we'll be updating those docs as we go.  Just concerned about the amount of technical debt we've brought over.

File: credential-registry/client/tob-web/src/themes/default/assets/recipes/start_a_restaurant.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
```
Comment (esune) [inline credential-registry/client/tob-web/src/themes/default/assets/recipes/start_a_restaurant.json]: I'm pretty sure we can remove this file, as it is not being used anymore.

File: credential-registry/client/tob-web/src/app/topic/form.component.ts [hunk 2]
```diff
@@ -129,7 +129,7 @@ export class TopicFormComponent implements OnInit, OnDestroy {
         if(this._loader.result.error)
           state = true;
         else
-          state = this.isLoaded('topic', 'related_to', 'related_from' /*, 'creds'*/);
+          state = this.isLoaded('topic', /*'related_to', */ 'related_to_relations' /*, 'related_from', 'creds'*/);
```
Comment (esune) [inline credential-registry/client/tob-web/src/app/topic/form.component.ts]: Should we have the commented block as a single piece at the end, for readability?

File: credential-registry/server/django-icat-api/api_v2/models/Credential.py [hunk 3]
```diff
@@ -42,6 +42,28 @@ def _cached(self, key, val):
...
+                remote_name = name
+        return remote_name
+
+    def get_remote_name(self):
+        names = self.all_names
+        has_assumed_name = False
+        remote_name = None
+        for name in names:
+            if name.type == 'entity_name_assumed':
+                has_assumed_name = True
```
Comment (esune) [inline credential-registry/server/django-icat-api/api_v2/models/Credential.py]: Are we missing an assignment of some kind here? 
The function seems to return the last name in the list for which the type is not `entity_name_assumed`: what happens if all names are of type `entity_name_assumed`?

**Comment by esune** [comment]
> 
> Documentation such as [starter-kits/credential-registry/server/README.md](https://github.com/bcgov/indy-catalyst/pull/228/files#diff-ed7a4e3b83c0bd627198a52d96ff5711) are referencing the existing OrgBook docs/code. I'm assuming we'll be updating those docs as we go. Just concerned about the amount of technical debt we've brought over.
I think we will probably need to do a pass once everything is sort of "stable" in this repo structure to reorganize links, but I agree. I logged [this issue](https://app.zenhub.com/workspaces/von---verifiable-organization-network-5adf53987ccbaa70597dbec0/issues/bcgov/indy-catalyst/229) to keep track of this.

**Comment by WadeBarnes** [comment]
I spun things up with these changes and everything seemed to be working as expected.

**Comment by esune** [comment]
@ianco: it looks like some commits may not be pgp-signed, and therefore merging is blocked...

**Pull request closed without merging**